frontend/src/components/personal/PreferencesPage.tsx

```diff
@@ -50,6 +51,11 @@ export default function PreferencesPage (): React.ReactElement {
     dispatch(changeMyData({nickname, email, id}))
   }, [dispatch, nickname, email, id])
 
+  const logOut = useCallback((e) => {
```

Comment:
Why does logout is on preference page? Can it be on every page?

Reply:
add logout to profile page.

Reply:
I thought about navbar on each page :)